persistence/src/main/java/cz/muni/fi/pa165/sportsclub/dao/CoachDaoImpl.java

```diff
@@ -0,0 +1,64 @@
+package cz.muni.fi.pa165.sportsclub.dao;
+
+import cz.muni.fi.pa165.sportsclub.entity.Coach;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+/**
+ * Implementation of CoachDAO
+ * @author Jan Cech
+ */
+@Repository
+@Transactional
+public class CoachDaoImpl implements CoachDao {
+
+    @PersistenceContext
+    private EntityManager entityManager;
```

Comment:
In every other file, the `EntityManager` is called `em`. Why not keep the tradition? 😜

Reply:
it seemed more apparent to me to use the whole word instead of abbreviation